ui/src/message_popup/linear_session/linear_session.jsx

```diff
@@ -14,10 +14,22 @@ export default React.createClass({
   propTypes: {
     questions: React.PropTypes.array.isRequired,
     questionEl: React.PropTypes.func.isRequired,
+    getNextQuestion: React.PropTypes.func,
     summaryEl: React.PropTypes.func.isRequired,
     onLogMessage: React.PropTypes.func.isRequired
   },
 
+  getDefaultProps() {
```

Comment:
Fixed. I'm not 100% sure I've done it the way you wanted though, so you can take a look and let me know.